backend/src/services/secret/secret-dal.ts

```diff
@@ -171,6 +171,50 @@ export const secretDALFactory = (db: TDbClient) => {
     }
   };
 
+  const findByFolderIds = async (folderIds: string[], userId?: string, tx?: Knex) => {
+    try {
+      // check if not uui then userId id is null (corner case because service token's ID is not UUI in effort to keep backwards compatibility from mongo)
+      if (userId && !uuidValidate(userId)) {
```

Comment:
Shouldn't this register as an error? The user wanted to pass in a user id but the id they put is is not valid so i think they should be notifed. What do you think?

Reply:
Hmm. We do it this way in other places, so either we change it everywhere, or we keep it consistent with the existing implementations I'd say